content/blog/open-sourcing-packetrusher-a-5g-core-performance-tester.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Open-sourcing PacketRusher: A 5G Core performance tester"
+date: 2023-12-13T09:44:43.865Z
+author: Valentin D'Emmanuele
+authorimage: /img/profile.png
+thumbnailimage: /img/thumbnail.png
+disable: false
+tags:
+  - opensource
+  - 5g
+  - gnodeb
+  - ue
+  - simulator
+---
+In the fast-evolving landscape of 5G technology, the demand for robust and efficient testing tools has never been higher. Enter PacketRusher, a cutting-edge 5G Core Network performance testing tool. As the lead developer behind this groundbreaking project, I am thrilled to share the power and potential PacketRusher holds in revolutionizing the way we test and optimize 5G networks.
```

Comment:
I will, however, give my opinion. I think here it would best be written as "5 G core network performance testing tool. But I could easily be persuaded in the other direction, especially since you do seem to be an expert in this particular subject :-).

Reply:
I agree!